deltachat-ffi/deltachat.h

```diff
@@ -2527,6 +2570,24 @@ void           dc_accounts_unref                (dc_accounts_t* accounts);
  */
 uint32_t       dc_accounts_add_account          (dc_accounts_t* accounts);
 
+/**
+ * Add a new closed account to the account manager.
+ * Internally, dc_context_new_closed() is called using a unique database-name
+ * in the directory specified at dc_accounts_new().
+ *
+ * If the function succeeds,
+ * dc_accounts_get_all() will return one more account
+ * and you can access the newly created account using dc_accounts_get_account().
+ * Moreover, the newly created account will be the selected one.
+ *
+ * @memberof dc_accounts_t
+ * @param accounts Account manager as created by dc_accounts_new().
+ * @return Account-id, use dc_accounts_get_account() to get the context object.
+ *     On errors, 0 is returned.
+ *
+ * @memberof dc_accounts_t
```

Comment:
`@memberof` is added two times to the doc comment of `dc_accounts_add_closed_account()`.

Reply:
Fixed